rcl/test/rcl/test_events.cpp

```diff
@@ -773,6 +765,21 @@ TEST_F(TestEventFixture, test_event_is_valid)
   tear_down_publisher_subscriber();
 }
 
+/*
+ * Test passing not init to take_event/get_handle
+ */
+TEST_F(TestEventFixture, test_invalid_to_take_event) {
```

Comment:
```suggestion
TEST_F(TestEventFixture, test_event_is_invalid) {
```

this is also testing `rcl_event_get_rmw_handle`, not only take.